### ISSUE TITLE
RADFIT write radfit.frame

### ISSUE THREAD

**Issue by janarig** [vegandevs/vegan#343]
Hello,
How can I save the results ("radfit.frame") from Radfit finction?
write.table(as.data.frame("radfit.frame")) does not work!
there is a message : "Error in as.data.frame.default(euk.rad) : 
  cannot coerce class ‘"radfit.frame"’ to a data.frame"

**Comment by gavinsimpson** [comment]
Can you give use an idea of what it is in the object that you want written to a text file? The object contains a lot of outputs, potentially for many models so it can't be converted to a data frame, but some elements of it could be if you tell us what you are trying to achieve?

If you just want to save the object out to disk, you could use `saveRDS("radfit.frame")` to serialize the R object to disk, but that is only of use if you want to read it back into another R session.

**Comment by janarig** [comment]
Thanks for the quick answer
I want to save the radfit result to compare the models' values for each site evaluated, 
some time ago I was able to save as table using 'write.table' function and coercing the radfit.frame with as.data.frame, but now it is not possible anymore, i have the error message.
like in the example:
data(BCI)
mod <- radfit(BCI[3:5,])
write.table(as.data.frame(mod), radfit_V9.txt)
Error in as.data.frame.default(euk.rad) : 
  cannot coerce class ‘"radfit.frame"’ to a data.frame

**Comment by gavinsimpson** [comment]
Right, but what were you expecting in the file? `radfit` is fitting 5 models to each of the three samples with your example code. As far as I can see in the git commit history, if the behaviour you described ever worked as described, it is from over a decade ago and must have been based on a form of `radfit` that bears little resemblance to what we have today.

What specifically do you need to export? You can't export the `glm` model objects behind each of the RAD fits for example. I'm not sure what you mean by "to compare the models' values"? Do you mean the model summaries that you get with `summary(mod)` or the curve fitted to the RDA that you get from `fitted(mod)`? Or something else?
